src/main/java/uk/gov/companieshouse/chd/order/api/validator/CreateItemRequestValidator.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.companieshouse.chd.order.api.validator;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.chd.order.api.dto.MissingImageDeliveriesDTO;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class CreateItemRequestValidator {
+
+    /**
+     * Validates the item provided, returning any errors found.
+     * @param item the item to be validated
+     * @return the errors found, which will be empty if the item is found to be valid
+     */
+    public List<String> getValidationErrors(final MissingImageDeliveriesDTO item) {
+        final List<String> errors = new ArrayList<>();
+        if (item.getId() == null || item.getId().isEmpty()) {
```

Comment:
This method could be simplified by adding a method to check that a string is null or empty and add a message to the errors list and call that.
e.g.

```
public List<String> getValidationErrors(final MissingImageDeliveriesDTO item) {
	List<String> errors = new ArrayList<>();
	checkString(item.getId(), errors, ID_MESSAGE);
	checkString(item.getCompanyName(), errors, COMPANY_NAME_MESSAGE);
}

private void checkString(String string, ArrayList<String> errors, String errorMessage) {
	if(string == null || string.isEmpty()){
		errors.add(errorMessage);
	}
}
```

Perhaps streamline it a bit more by having the errors list as a global and initialise it when calling the getValidationErrors method

Reply:
implemented